ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/clientlibs/editor/utils/utils.js

```diff
@@ -167,25 +167,26 @@
             _managePatternDynamicBehaviour();
             patternComponent.addEventListener("change", _managePatternDynamicBehaviour );
             function _managePatternDynamicBehaviour() {
-                var displayPatternSelectedValue = patternComponent.selectedItem.innerHTML;
+                // FORMS-12822, below the pattern was compared based on the name rather than the value ("No Pattern" instead of ####.####) which was creating issue in other languages therefore now it has changed to value.
+                var displayPatternSelectedValue = patternComponent.selectedItem.value;
                 var patternComponentOptionsNodeList=patternComponent.querySelectorAll('coral-select-item');
                 if(patternComponentOptionsNodeList.length<=2 ){
-                  //there are 2 default options, "Select" and "custom".
+                    //there are 2 default options, "Select" and "custom".
                     // For this dropdown to be visible it should have atleast one other option
                     var patternComponentParentDiv=patternComponent.closest("div");
                     patternComponentParentDiv.setAttribute("hidden", true);
                 }else {
                     var displayFormatParentDiv=formatComponent.closest("div");
```

Comment:
Since you are already changing the code, can you move this to ES6 style with let and const.